internal/config/supervisor/types.go

```diff
@@ -7,11 +7,12 @@ import "go.pinniped.dev/internal/plog"
 
 // Config contains knobs to setup an instance of the Pinniped Supervisor.
 type Config struct {
-	APIGroupSuffix *string           `json:"apiGroupSuffix,omitempty"`
-	Labels         map[string]string `json:"labels"`
-	NamesConfig    NamesConfigSpec   `json:"names"`
-	LogLevel       plog.LogLevel     `json:"logLevel"`
-	Endpoints      *Endpoints        `json:"endpoints"`
+	APIGroupSuffix    *string           `json:"apiGroupSuffix,omitempty"`
+	Labels            map[string]string `json:"labels"`
+	NamesConfig       NamesConfigSpec   `json:"names"`
+	LogLevel          plog.LogLevel     `json:"logLevel"`
+	Endpoints         *Endpoints        `json:"endpoints"`
+	AllowExternalHTTP string            `json:"insecure_accept_external_unencrypted_http_requests"`
```

Comment:
For points 2 and 3, updated in 51c527a965a743f89347420e6a5fe075a59ef3f0.

For you first point about moving it inside the `endpoints` object, that's certainly possible. I had made it a top-level value in values.yaml because we intend to delete it in a future release, and I thought it would be easier to delete a top-level option than a value embedded inside a map that is opaque to ytt. For example, if we wanted to introduce an error for anyone who tries to use the value with ytt after it is deleted (similar to what this PR does for the deleted `service_http_*` values) then it will be easier if it is a top-level value.